x/programs/cmd/simulator/cmd/request.go

```diff
@@ -32,28 +32,28 @@ var _ Cmd = (*runCmd)(nil)
 type runCmd struct {
 	cmd *argparse.Command
 
-	lastStep *int
-	file     *string
-	planStep *string
+	// counts the number of requests made
+	numRequests *int
```

Comment:
huge nit (you can just resolve this if you disagree)

`requestCount`? `num` isn't descriptive, that's what the type-system is for....

Reply:
reason i had `numRequests` over `requests` was because there's already a `request` field with a different type. And `requests` in our rust side has a different type.